examples/TTOpen-service.json

```diff
@@ -0,0 +1,10 @@
+{
+"services": [
+   {
+       "resource": "/iot/d",
+       "apikey": "pipopi",
+       "cbroker" : "http://127.0.0.1:10026",
+       "entity_type": "ThinkingThing"
+   }
+]
+}
```

Comment:
The indentation (use of tabs or blank spaces or whatever it is called :) ) of this file is not the standard one. Just in case you want to update it to:
```json
{
  "services": [
    {
      "resource": "/iot/d",
      "apikey": "pipopi",
      "cbroker" : "http://127.0.0.1:10026",
      "entity_type": "ThinkingThing"
    }
  ]
}
```